File: 프로그래머스/Lv.0/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        int even = 0;
        int odd = 0;

        for (int num : num_list) {
            if (num % 2 == 0) {
                even *= 10;
                even += num;
            } else {
                odd *= 10;
                odd += num;
            }
        }

        return even + odd;
    }
}

File: 프로그래머스/Lv.0/181928. 이어 붙인 수/이어 붙인 수.kt
class Solution {
    fun solution(num_list: IntArray): Int {
        var even = 0
        var odd = 0

        for (num in num_list) {
            if (num % 2 == 0) {
                even *= 10
                even += num
            } else {
                odd *= 10
                odd += num
            }
        }

        return even + odd
    }
}